simbad/mr/__init__.py

```diff
@@ -557,6 +557,30 @@ def summarize(self, csv_file):
                          csv_file=csv_file, columns=columns)
 
 
+def best_result_csv(f, run_type):
```

Comment:
I think this function is better placed in the `simbad/util/__init__.py` file. I think its definition should look something like 
```
def result_by_score_from_csv(file, score, ascending=True):
    df = pd.read_csv(file)
    df.sort_values(score, ascending=ascending, inplace=True)
    return df.loc[0, ["pdb", score]].tolist()
```
The `run_type` seems redundant and should probably added outside of this function.

Reply:
I've added in this suggestion in 8c92bb1